tests/ssl-opt.sh

```diff
@@ -1163,6 +1163,7 @@ run_test    "Encrypt then MAC: client disabled, server enabled" \
             -S "using encrypt then mac"
 
 requires_config_enabled MBEDTLS_SSL_PROTO_SSL3
+requires_config_disabled MBEDTLS_DEPRECATED_REMOVED
```

Comment:
Again, I don't think we should need to do that. I'd much rather just add a check in `check_config.h`.